docroot/sites/registrar.uiowa.edu/modules/registrar_core/registrar_core.module

```diff
@@ -14,5 +16,30 @@ function registrar_core_preprocess_block(&$variables) {
     case 'academic_calendar_block':
       uiowa_core_set_attributes($variables, 'grid--twocol--25-75');
       break;
+
+    // Add gpa calculator styles.
+    case 'gpa_calculator':
+      $variables['#attached']['library'][] = 'registrar_core/gpa-calculator';
+      break;
+  }
+}
+
+/**
+ * Implements hook_form_alter().
+ */
+function registrar_core_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if ($form_id === 'gpa_calculator_form') {
+    $form['gpa_row']['actions']['add_row']['#attributes']['class'][] = 'button--small';
+    $form['gpa_row']['actions']['add_row']['#attributes']['class'][] = 'button--full-width';
+    $form['#submit'][] = 'registrar_core_post_submit';
   }
 }
+
+/**
+ * Triggers after submit to change the way that the grade output is generated.
+ */
+function registrar_core_post_submit(&$form, FormStateInterface $form_state) {
+  $form['#attached']['library'][] = 'uids_base/callout';
```

Comment:
Does this need to trigger each submit or can it just load as part of the form alter?

Reply:
We might be able to load on the form alter, but i remember there being some odd things in conjunction with the block regenerating. I can test it in the form alter and get back to this.